### ISSUE TITLE
psite HDF5 file is saved in the wrong location

### ISSUE THREAD

**Issue by Roleren** [xryanglab/RiboCode#31]
Correct me if I am wrong, but currently, the hdf5 file with psite information is saved in working dir and not in defined output-dir, this makes it fail if output-dir is not the working directory for the call. 

This was tested on the latest version through Conda.

**Comment by zhengtaoxiao** [comment]
Yes, it would be saved in working dir. RiboCode only provides an option for defining the outputs of annotation files. The hdf5 file will be named according to the information in the config file.

**Comment by Roleren** [comment]
There should be an advanced FAQ where this is stated, together with the fact you can drastically speed up RiboCode if you input a hdf5 file. Has to tinker a lot to get this working and understand it all, nice for others to know :)
